Sources/Styleguide/PointFreeBaseStyles.swift

```diff
@@ -399,12 +416,21 @@ private let _navBar = CssSelector.class("pf-navbar")
 private let navBarStyles =
   ((_navBar ** a) | (_navBar ** a & .pseudo(.link))) % color(.other("#fff"))
 
-private let baseButtonClass = CssSelector.class("btn")
-private let baseButtonStyles: Stylesheet =
-  (baseButtonClass & .pseudo(.hover)) % darken1
-    <> (a & .pseudo(.active) & baseButtonClass) % darken3
-    <> (a & .pseudo(.link) & baseButtonClass) % key("text-decoration", "none")
-    <> baseButtonClass % padding(topBottom: .rem(0.75))
+private let baseButtonStyles =
+  baseNormalButtonStyles
+    <> baseUnderlineButtonStyles
+
+private let baseNormalButtonClass = CssSelector.class("btn-normal")
+private let baseNormalButtonStyles: Stylesheet =
+  (baseNormalButtonClass & .pseudo(.hover)) % darken1
+    <> (a & .pseudo(.active) & baseNormalButtonClass) % darken3
+    <> (a & .pseudo(.link) & baseNormalButtonClass) % key("text-decoration", "none")
+    <> baseNormalButtonClass % padding(topBottom: .rem(0.75))
+
+private let baseUnderlineButtonClass = CssSelector.class("btn-outline")
+private let baseUnderlineButtonStyles: Stylesheet =
+  (baseUnderlineButtonClass & .pseudo(.hover)) % key("text-decoration", "none !important")
+    <> baseUnderlineButtonClass % key("text-decoration", "underline !important")
```

Comment:
ok this `!important` isnt ideal, but seemed necessary due to us making all `a`'s have no underline. can revisit easily since we have so many tests!